Tests/MongoSwiftTests/MongoClientTests.swift

```diff
@@ -22,16 +22,16 @@ final class MongoClientTests: MongoSwiftTestCase {
         let connectionString = MongoSwiftTestCase.connStr
         var error = bson_error_t()
         guard let uri = mongoc_uri_new_with_error(connectionString, &error) else {
-            throw MongoError.invalidUri(message: toErrorString(error))
+            throw parseMongocError(error: error)
         }
 
         guard let client_t = mongoc_client_new_from_uri(uri) else {
-            throw MongoError.invalidClient()
+            throw RuntimeError.internalError(message: "creating client from uri failed")
```

Comment:
Here we use a different error than the TLS one above

Reply:
This is in a test file so I didn't really bother unifying them, but for consistency it makes sense to. Fixed.